platform/ui/src/components/LineChart/LineChart.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useEffect, useLayoutEffect, useState, useRef } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { lineChart } from './d3LineChart';
+import * as d3 from 'd3';
+import './LineChart.css';
+
+const LineChart = ({
+  width: widthProp,
+  height: heightProp,
+  axis,
+  series,
```

Comment:
I'm surpised there is a prop called series here... is this DICOM series?

Reply:
No, chart series